<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterPartner1</class>
 <widget class="QWidget" name="FilterPartner1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="WdtContainer" name="wc" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="C5LineEdit" name="lineEdit_3">
        <property name="lvalue" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="C5LineEdit" name="lineEdit_2">
        <property name="lvalue" stdset="0">
         <string>ticket_type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ticket</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="C5LineEdit" name="leTicket">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>state</string>
        </property>
        <property name="lkey" stdset="0">
         <string>ticket_type</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>352</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="C5LineEdit" name="leFuel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>fuel</string>
        </property>
        <property name="lkey" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fuel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="C5LineEdit" name="lePartner">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>partner</string>
        </property>
        <property name="lkey" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="C5LineEdit" name="lineEdit_4">
        <property name="lvalue" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WdtContainer</class>
   <extends>QWidget</extends>
   <header location="global">wdtcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFuel</tabstop>
  <tabstop>leTicket</tabstop>
  <tabstop>lePartner</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
